editor/src/components/canvas/dom-walker.ts

```diff
@@ -528,38 +607,16 @@ function collectMetadata(
       )
     })
   if (shouldCollect && pathsForElement.length > 0) {
```

Comment:
You can remove the `pathsForElement.length > 0` check here now

Reply:
done